src/components/Vote.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { patchArticleVotes } from "../../api"
+
+export default function Vote({id, articleVotes}) {
+    const [newVote, setNewVote] = useState(Number(articleVotes));
+    const [errorMsg, setErrorMsg] = useState(null);
+    
+    function handleVote(addToVote) {
+        setNewVote((voteToChange) => voteToChange + addToVote);
+
+        patchArticleVotes(id, {
+            inc_votes: addToVote,
+            article_id: id
+        })
+        .then((result) => {
+            setNewVote(result.article.votes)
```

Comment:
as we've already optimistically set the new votes above, we don't actually need this then block - the ui has already been updated! :)